packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -83,6 +83,26 @@ library LibVerifying {
         });
     }
 
+    function resetGenesisHash(TaikoData.State storage _state, bytes32 _genesisBlockHash) internal {
```

Comment:
we do not revert in this function so that if we have another reinitializer, the sequence number won't mess up.